src/meltano/cli/install.py

```diff
@@ -52,24 +51,18 @@ def install(project, plugin_type, plugin_name, clean, parallelism):
 
     plugins_service = ProjectPluginsService(project)
 
-    if plugin_type:
-        try:
+    try:
+        if plugin_type:
             plugin_type = PluginType.from_cli_argument(plugin_type)
-        except ValueError:
-            # if we fail because plugin_type is not valid we have no plugins to instrument
-            tracker.track_command_event(cli_tracking.STARTED)
-            tracker.track_command_event(cli_tracking.ABORTED)
-            raise
-        plugins = plugins_service.get_plugins_of_type(plugin_type)
-        if plugin_name:
-            plugins = [plugin for plugin in plugins if plugin.name in plugin_name]
-    else:
-        try:
+            plugins = plugins_service.get_plugins_of_type(plugin_type)
+            if plugin_name:
+                plugins = [plugin for plugin in plugins if plugin.name in plugin_name]
+        else:
             plugins = list(plugins_service.plugins())
-        except PluginNotFoundError:
-            tracker.track_command_event(cli_tracking.STARTED)
-            tracker.track_command_event(cli_tracking.ABORTED)
-            raise
+    except Exception:
```

Comment:
@pandemicsyn I'm not too familiar with the new tracking setup, but do we get information about which exception was raised here?

Reply:
@edgarrmondragon theres also this pair of issues:

Refactor command tracking event/context to be initialized earlier - #6041
Collect process exit events with Snowplow - #5986 

Those two issue will help clean up the start/end of the tracker life cycles.